source/features/table-input.css

```diff
@@ -30,12 +30,6 @@
 	}
 }
 
-/** TODO: Drop JS-based version after :has() has full support */
-.rgh-table-input:hover .selected div {
```

Comment:
No full support, but the feature now _requires_ `:has()`, so I'm dropping the fallback.